software/wireshark/en.md

```diff
@@ -0,0 +1,24 @@
++++
+title = "Wireshark"
+lastmod = "2018-04-27T21:20:05+02:00"
++++
+# Installation
+
+Wireshark can be installed either from the Software Center or via terminal:
+
+``` bash
+sudo eopkg it wireshark
+```
+
+# Use wireshark as a normal user
```

Comment:
Header level 1 should not be used for this. Also should be: `as a non-root user` instead of `as a normal user`